store-api-server/src/payment/controller/payment.controller.ts

```diff
@@ -150,4 +150,20 @@ export class PaymentController {
       );
     }
   }
+
+  @Get('/is-paid-simplex/:paymentId')
```

Comment:
Actually it is not about exposing an endpoint that tells consumers of our API whether the simplex payment is succeeded. We have our own status tracking in our database, which ultimately determines what NFTs are returned in the various profile page endpoints (doesn't matter which payment provider was used).

So the point is rather to just update our internal tracked status accordingly, which you do correctly here: https://github.com/tzConnectBerlin/kanvas/pull/175/files/c9dc7a0cdbc30f26a18c27ec586d5b9c3abbae9d..2bcbac53e149195638495adfb4d7b38c5047e899#diff-6a644324b05d3f328b0e27a27c789a907de6c5f566a10482bbd20ac74d021e3fR193.

But it should be something that is not triggered from an external API call. Instead, make it a polling with a Cron (see for example this: https://github.com/umitduran/motion_kanvas/blob/feat/simplexapiintegration/store-api-server/src/payment/service/payment.service.ts#L848). And it should of course be done then in each poll iteration for all `paymentId` that are not finalized yet.

Reply:
Ok, actually next step will be job scheduler. But The system does not require any req from FE to update paymentstatus I got it. I will prepare job